go/test/endtoend/vreplication/vreplication_test.go

```diff
@@ -112,6 +112,95 @@ func throttlerCheckSelf(tablet *cluster.VttabletProcess, app string) (resp *http
 	return resp, respBody, err
 }
 
+// TestVreplicationDDLHandling tests the DDL handling in
```

Comment:
Do we need an end-to-end test here?  We have been discussing about reducing CI load by being careful about reviewing existing e2e tests where unit tests coverage is possible.